sql_normalization/sql_normalization.md

```diff
@@ -0,0 +1,370 @@
+<!--
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+module_template_version: 3.0.0
+language: en
+narrator: US English Female
+title: SQL Normalization
+comment: Learn about the concept of normalization and why it's important for SQL.
+long_description: Usually, data in a SQL database is organized into multiple interrelated tables with as little data repetition as possible. This module teaches underlying data considerations and explains how data is organized in SQL by introducing the concepts of one-to-many data relationships and normalization.
+estimated_time: 40 minutes
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Explain the significance of "one to many" data relationships and how these relationships affect data organization
+- Describe how a normalized database is typically organized
+- Explain how data can be linked between tables and define "primary keys" and "foreign keys"
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+-->
+
+# SQL Normalization
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites**  Learners should have experience writing SQL code on single tables.  If you have successfully used a "SELECT... FROM... WHERE" SQL statement on a single table, this is sufficient for this module. If you need to develop basic SQL fluency we recommend our module [SQL Basics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/sql_basics/sql_basics.md).  
```

Comment:
```suggestion
**Pre-requisites**  Learners should have experience working with data in tables.  This could included working with .csv files, SQL databases, R data frames, REDCap instruments, or other ways that data can be collected in tables. 
```